delphi/translators/for2py/preprocessor.py

```diff
@@ -226,11 +226,8 @@ def get_preprocessed_lines_from_file(infile):
         lines = preprocess_lines(inputLines, infile)
         return lines
 
-def create_preprocessed_file(infile, outfile=None, forModLogGen=False):
+def create_preprocessed_file(infile, forModLogGen=False):
```

Comment:
@hlim1 Since this function does not create a file anymore, could you please rename it to something else? Or even better - combine the `get_preprocessed_lines_from_file` and `create_preprocessed_file` functions into one function, since they are virtually identical.

Reply:
@adarshp I've merged two functions and only left `get_preprocessed_lines_from_file`.